optimum/intel/openvino/trainer.py

```diff
@@ -749,9 +751,12 @@ def _should_apply_pruning_transform(self) -> bool:
 
     def _reshape_ir(self, ov_model: openvino.runtime.Model, static_shape: bool) -> openvino.runtime.Model:
         new_input_cfg = dict()
+        input_name_vs_shape = {item["keyword"]: item["sample_size"] for item in self.ov_config.input_info}
```

Comment:
Is there a possibility that `self.ov_config.input_info` is not defined at this point?